x-pack/plugins/observability/server/types/models/slo.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as t from 'io-ts';
+
+import { indicatorSchema, rollingTimeWindowSchema } from '../schema';
+
+const baseSLOSchema = t.type({
+  id: t.string,
+  name: t.string,
+  description: t.string,
+  indicator: indicatorSchema,
+  time_window: rollingTimeWindowSchema,
+  budgeting_method: t.literal('occurrences'),
+  objective: t.type({
+    target: t.number,
+  }),
+  settings: t.partial({
+    destination_index: t.string,
+  }),
+});
+
+const storedSLOSchema = t.intersection([
+  baseSLOSchema,
+  t.type({ created_at: t.string, updated_at: t.string }),
```

Comment:
So we will not return `created_at` and `updated_at` information to use in UI, right?

Reply:
At the moment no, but we might want to surface this later and change the types